src/view/components/molecules/MetamaskConnect.tsx

```diff
@@ -48,12 +47,9 @@ const MetamaskConnect = (): JSX.Element => {
     return (
         <header>
             {walletAddress ? (
-                <ConnectBtn
-                    disconnect={onClickDisconnect}
-                    walletAddress={walletAddress}
-                />
+                <Btn click={onClickDisconnect} text={`Account : ${walletAddress}`} />
```

Comment:
Do you need to show a difference in UI when it's being used for connect/disconnect. If yes, use a button variant